core/base/AutoreleasePool.h

```diff
@@ -122,6 +122,8 @@ class AX_DLL AutoreleasePool
      */
     void dump();
 
+    void setNeedPop(bool needPop);
```

Comment:
Please explain the purpose of this method, and at least add a comment above it with the relevant info.